integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -320,6 +322,37 @@ public void testAdminT3ChannelWithJMS(Domain domain) throws Exception {
     logger.info("Done - testAdminT3ChannelWithJMS");
   }
 
+  /**
+   * Verify Load Balancing by deploying and invoking webservicebapp
+   *
+   * @throws Exception
+   */
+  public void testWSLoadBalancing(Domain domain) throws Exception {
+    logger.info("Inside testWSLoadBalancing");
+    TestUtils.renewK8sClusterLease(getProjectRoot(), getLeaseId());
+    Map<String, Object> domainMap = domain.getDomainMap();
+
+    ExecResult result =
```

Comment:
you can call TestUtils.kubectlexec() here and change kubectlexec() to write both stderr and stdout like below so that method checks for the exitValue.
if (result.exitValue() != 0) {
      logger.info(
          "Command "
              + cmd
              + " failed with stderr = "
              + result.stderr()
              + " \n stdout = "
              + result.stdout());
      throw new RuntimeException(
          "FAILURE: Command "
              + cmd
              + " failed with stderr = "
              + result.stderr()
              + " \n stdout = "
              + result.stdout());
}

Reply:
replaced and cleaned up